test/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from unittest.mock import patch
+from torchao.utils import torch_version_at_least
+
+class TestTorchVersionAtLeast(unittest.TestCase):
+    def test_torch_version_at_least(self):
+        test_cases = [
+            ("2.5.0a0+git9f17037", "2.5.0", True),
```

Comment:
I'm confused, as I thought this should be False instead.
shouldn't 2.5.0a0 and 2.5.0.dev be <2.5.0 but >2.4?
I might misunderstand `torch_version_at_least` usage though

Reply:
Strictly speaking this is an internal only utility that we leverage in case torch makes certain breaking changes or adds new features upstream. So I can't think of many cases where we'd do one thing with 2.5 nightlies vs 2.5 official releases.

Reply:
understood.